linker/shared/src/main/scala/org/scalajs/linker/backend/wasmemitter/VarGen.scala

```diff
@@ -292,6 +292,7 @@ object VarGen {
       case object vtable extends FieldID
       case object itables extends FieldID
       case object arrayUnderlying extends FieldID
+      final case class itableSlot(i: Int) extends FieldID
```

Comment:
This should be in a separate `object itablesStruct` namespace. `objStruct` contains field IDs for fiels of structs that extend `jl.Object`. The `itables` struct is a separate struct type, like `reflectiveProxy` or `typeData`.